zerver/views/user_groups.py

```diff
@@ -124,6 +176,7 @@ def add_members_to_group_backend(
     user_group = access_user_group_by_id(user_group_id, user_profile)
     user_profiles = user_ids_to_users(members, user_profile.realm)
     existing_member_ids = set(get_direct_memberships_of_users(user_group, user_profiles))
+    acting_user = user_profile
```

Comment:
There is no need of this extra variable.

Reply:
Removing this variable causes a bug beacuse the variable `user_profile` is used for iterating inside a loop in the function.
So after iteration is completed `user_profile` is no longer the same person which made request.

If we need to remove this variable then variable used for iterating loop should be changed.

Reply:
Ok. thanks! We can leave this as it is.

Reply:
I think it's generally better to address this sort of thing by using a different loop variable -- redefining the meaning of a variable midfunction leads to confusing code and that leads to bugs. Something of the form `foo_user` should work great.

Reply:
Yes, I have done the changes in a separate commit.